install/_lib.sh

```diff
@@ -6,14 +6,13 @@ log_file="sentry_install_log-`date +'%Y-%m-%d_%H-%M-%S'`.txt"
 exec &> >(tee -a "$log_file")
 
 # Work from /install/ for install.sh, project root otherwise
-if [[ "$(basename $0)" = "install.sh" ]]; then
+if [[ "$(basename $0)" = "install.sh" || "$(basename $0)" = "test.sh" ]]; then
   cd "$(dirname $0)/install/"
 else
   cd "$(dirname $0)"  # assume we're a *-test.sh script
 fi
 
-_ENV="$(realpath ./.env)"
-
+_ENV="$(realpath ../.env)"
```

Comment:
This is still a problem, though. Right? Don't we need the first commit that varies this with the base dir?

Reply:
Nope, the logic there was valid :D

Reply:
Got it, because dirname of $0 is `.../install/` if we assume it's a `*-test.sh` script. 👍